File: app/src/main/java/com/example/linqing/zhihumvp/list/ListContract.java
package com.example.linqing.zhihumvp.list;

import com.example.linqing.zhihumvp.entitybean.ItemBean;

import java.util.List;

public interface ListContract {
    interface ListView{
        void updateList(ItemBean storiesBeanList);

        void refreshList();//init list
        void getItemData(ItemBean itemBean);
        void notifyChange();



    }
    interface ListPresenterInter{
        void getListData();
        void getPrevious(String date);


    }

}

File: app/src/main/java/com/example/linqing/zhihumvp/list/ListContract.kt
package com.example.linqing.zhihumvp.list

import com.example.linqing.zhihumvp.entitybean.ItemBean

interface ListContract {
    interface ListView {
        fun updateList(storiesBeanList: ItemBean)

        fun refreshList() //init list
        fun getItemData(itemBean: ItemBean)
        fun notifyChange()


    }

    interface ListPresenterInter {
        fun getListData()
        fun getPrevious(date: String)


    }

}
